packages/portal-proto/src/components/Image.tsx

```diff
@@ -1,5 +1,5 @@
 import getConfig from "next/config";
-import NextImage, { ImageProps } from "next/image";
+import NextImage, { ImageProps } from "next/legacy/image";
```

Comment:
Is there a reason we need to import from this legacy package instead of switching to the new image component?

Reply:
Yes, I can tackle it in another ticket. Here are some of the changes that Craig referred.
https://nextjs.org/docs/pages/api-reference/components/image-legacy#comparison